### PR TITLE
Fix: progam modem fw on 9131 and 9151

### PR DESCRIPTION
**Pull request by kylebonnici**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 -   The **Enable MCUboot** toggle reflects the MCUboot trait of the connected
     device.
+-   Programming modem firmware with external JLink is now possible on nrf9131
+    and nrf9151
```
Comment (greg-fer) [inline Changelog.md:13]:
```suggestion
-   Programming modem firmware with an external J-Link is now possible on nRF9131 DK
    and nRF9151 DK.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]
Punctuation ticker: 40 NOK

File: Changelog.md (hunk 1 above)
Comment (kylebonnici) [reply Changelog.md:13]: They do not have to be DK a custom kit with that SoC will use an external JLink and will have same issue

```suggestion
-   Programming modem firmware with an external J-Link is now possible on nrf9131
    and nrf9151
```

Comment (kylebonnici) [reply Changelog.md:13]: @greg-fer  any commends on the above?

Comment (greg-fer) [reply Changelog.md:13]: All good.

**Pull request closed without merging**